Test/Microsoft.ApplicationInsights.Test/Shared/Channel/TransmissionTest.cs

```diff
@@ -255,17 +255,6 @@ public async Task SendAsyncHandlesTimeout()
             {
                 int clientTimeoutInMillisecs = 1;
 
-                //ARRANGE
```

Comment:
why is this change required?

Reply:
simply clean up as this was dead code that wasn't used in a test that failed.

Reply:
It intention was to ensure that the timeout occurs when used properly:
using (var fakeHttpClient = new HttpClient(handler )).

But since tests are passing consistently, we can remove.